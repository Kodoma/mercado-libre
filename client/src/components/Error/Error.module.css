.errorContainer {
    margin: 0 auto;
    position: relative;
    display: flex;
    flex-direction: column;
    align-content: center;
    align-items: center;
    text-align: center;
}
