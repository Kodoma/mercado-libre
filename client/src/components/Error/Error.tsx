import React from 'react';
import styles from './Error.module.css';
import { Typography } from '@material-ui/core';
import NotFound from '../../../public/images/svg/notfound.svg';

const ErrorComponent = () => {
    return (
        <div className={styles.errorContainer}>
            <NotFound />
            <Typography variant={'h6'}>Parece que esta p√°gina no existe</Typography>
        </div>
    );
};

export default React.memo(ErrorComponent);
