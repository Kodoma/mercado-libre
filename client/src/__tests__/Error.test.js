import React from 'react';
import { shallow } from 'enzyme';
import Error from "../components/Error/Error";

describe('__tests__ Search', () => {
    it('should render my Search component', () => {
        const component = shallow(<Error />);
        expect(component.getElements()).toMatchSnapshot();
    });
});
