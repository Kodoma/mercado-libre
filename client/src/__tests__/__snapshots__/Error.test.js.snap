// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests Search should render my Search component 1`] = `
Array [
  <div
    className="errorContainer"
  >
    <NotFound
      height="154"
      width="251"
      xmlns="http://www.w3.org/2000/svg"
    />
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Parece que esta p√°gina no existe
    </WithStyles(ForwardRef(Typography))>
  </div>,
]
`;
